src/handwritten/_grpc_time.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timezone
+from hightime import datetime as ht_datetime
+
+from google.protobuf.internal.well_known_types import Timestamp as GrpcTimestamp
+
+
+_NS_PER_S = 10**9
+_NS_PER_US = 10**3
+
+_YS_PER_US = 10**18
+_YS_PER_NS = 10**15
+_YS_PER_FS = 10**9
+
+
+def convert_time_to_timestamp(dt, ts):
```

Comment:
Sorry to bring this up late, but consider adding type hints in order to document the signature. I expected this to return a timestamp object, but it mutates the one you pass in, probably because protobuf in Python doesn't allow reassigning child messages after construction.

```suggestion
def convert_time_to_timestamp(dt: ht_datetime, ts: GrpcTimestamp) -> None:
```